sys-block/open-iscsi/open-iscsi-2.0.874-r1.ebuild

```diff
@@ -113,9 +113,9 @@ pkg_postinst() {
 	in='/etc/iscsi/initiatorname.iscsi'
 	if [ ! -f "${EROOT}${in}" -a -f "${EROOT}${in}.example" ]; then
 		{
-		  cat "${EROOT}${in}.example" 
+		  cat "${EROOT}${in}.example"
```

Comment:
Agreed, it was just an opportunity to squash a very minor repoman-reported whitespace issue.